examples/meta/CMakeLists.txt

```diff
@@ -27,18 +27,25 @@ FOREACH(META_EXAMPLE ${META_EXAMPLES})
     LIST(APPEND EXAMPLE_LISTINGS ${CMAKE_CURRENT_BINARY_DIR}/r/${BASENAME}.R)
     LIST(APPEND EXAMPLE_LISTINGS ${CMAKE_CURRENT_BINARY_DIR}/lua/${BASENAME}.lua)
 
+	# Set generate.py flags
```

Comment:
indentation not respecting the previous one, makes reviewing much harder for me btw